distmaker/distmaker.sh

```diff
@@ -178,11 +179,15 @@ case $1 in
   ;;
 
   # JOOMLA PHP5
-  j5|Joomla)
+  Joomla)
   echo; echo "Generating Joomla PHP5 module"; echo;
   J5PACK=1
   ;;
 
+  j5|Joomla5)
+  echo; echo "Generating Joomla 5 PHP Module"; echo;
+  JOOMLA5PACK=1
+
```

Comment:
@seamuslee001 I've just implemented a label `run-distmaker` (*which generates tarballs from a PR*) and tried here a test-case. If you check results on `CiviCRM-Distmaker-PR`, you should be able to see this console error:

```
  j5|Joomla5)
  echo; echo "Generating Joomla 5 PHP Module"; echo;
  JOOMLA5PACK=1

  # WORDPRESS PHP5
  wp5|WordPress)
./distmaker.sh: line 192: syntax error near unexpected token `)'
[chrome-headless] Kill process (pid=4474, sig=15)
[mysql] Kill process (pid=4258, sig=15)
```

It's missing `;;` at the end of Joomla5 block.

(Aside: Hmm, after fixing that, there's another issue... tester is [not going to run the new `Joomla5` target](https://github.com/civicrm/civicrm-buildkit/blob/11856f8f72f5b24919b358c086cea5f72ab4ed2d/src/jobs/CiviCRM-Distmaker.job#L116-L124)... )